handel/src/services/ecs-fargate/params-schema.json

```diff
@@ -0,0 +1,300 @@
+                        }
+                    },
+                    "links": {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "errorMessage": "The 'links' parameter must be a string"
+                        }
+                    },
+                    "routing": {
+                        "type": "object",
+                        "properties": {
+                            "base_path": {
+                                "type": "string",
+                                "errorMessage": "The 'base_path' field of the 'routing' section must be a string"
+                            },
+                            "health_check_path": {
+                                "type": "string",
+                                "errorMessage": "The 'health_check_path' field of the 'routing' section must be a string"
+                            }
+                        },
+                        "required": [
+                            "base_path"
+                        ],
+                        "additionalProperties": false,
+                        "errorMessage": {
+                            "required": {
+                                "base_path": "The 'base_path' parameter is required in the 'routing' section"
+                            },
+                            "additionalProperties": "Invalid/Unknown property specified in the 'routing' section"
+                        }
+                    },
+                    "environment_variables": {
+                        "type": "object",
+                        "patternProperties": {
+                            "^.*$": {
+                                "anyOf": [
+                                    {"type": "string"},
+                                    {"type": "number"}
+                                ]
+                            }
+                        },
+                        "errorMessage": "The 'environmet_variables' element must contain 1 or more simple key/value pairs where the values are strings or numbers",
+                        "additionalProperties": false
+                    }
+                },
+                "required": [
+                    "name"
+                ],
+                "additionalProperties": false,
+                "errorMessage": {
+                    "required": {
+                        "name": "The 'name' parameter is required in each container in the 'containers' section"
+                    },
+                    "additionalProperties": "Invalid/Unknown property specified in the 'containers' section"
+                }
+            }
+        },
+        "auto_scaling": {
+            "type": "object",
+            "properties": {
+                "min_tasks": {
+                    "type": "integer",
+                    "errorMessage": "The 'min_tasks' field must be a string in the 'auto_scaling' section"
```

Comment:
The error message here and for 'max_tasks' should say it should be an integer instead of a string.